### DISCUSSION TITLE
Quantsapp Credentials

### DISCUSSION THREAD

**Discussion by amitvj24** [TechfaneTechnologies/QtsApp#4] [Q&A] [answered]
As per your Instructions, we have to save Username and Password to .env.secret file.
I do not have QuantsApp account yet. But they are offering 7-days free trial.

Post 7 days of Free Trial, as I would ideally have same username and password but if I don't subscribe to any of their plan, will I still be able to get the data using this module?

**Comment by MooneDrJune** [comment, marked as answer]
> As per your Instructions, we have to save Username and Password to .env.secret file. I do not have QuantsApp account yet. But they are offering 7-days free trial.
> 
> Post 7 days of Free Trial, as I would ideally have same username and password but if I don't subscribe to any of their plan, will I still be able to get the data using this module?

Yes, This Module works with the free plan.
